wsmaster/che-core-api-devfile/README.md

```diff
@@ -136,6 +136,44 @@ So, alternatively, if you need to post devfile with such tools to REST API, cont
               ... etc
 ```
 
+As with `dockerimage` tool described below, it is possible to override the entrypoint of the
+containers contained in the Kubernetes/Openshift list using the `command` and `args` properties (as
+[understood](https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/#notes)
+by Kubernetes). Of course, there can be more containers in the list (contained in pods or pod
+templates of deployments) and so there needs to be a way of selecting which containers to apply the
+entrypoint changes to.
+
+The entrypoints can be defined for example like this:
+
+```yaml
+...
+  tools:
+    - name: appDeployment
+      type: kubernetes
+      local: app-deployment.yaml
+      entrypoints:
+      - pod: mysqlServer
+        command: ['sleep']
+        args: ['infinity']
+      - deploymentSelector: prometheus
```

Comment:
Please make sure that examples are up to date. `parentSelector` should be instead of `deploymentSelector`